pkg/ccl/streamingccl/streamproducer/producer_job.go

```diff
@@ -44,12 +44,17 @@ func makeProducerJobRecord(
 	timeout time.Duration,
 	user username.SQLUsername,
 	ptsID uuid.UUID,
+	postCutoverRetention bool,
 ) jobs.Record {
 	tenantID := tenantInfo.ID
 	tenantName := tenantInfo.Name
+	description := fmt.Sprintf("Physical replication stream producer for %q (%d)", tenantName, tenantID)
```

Comment:
Maybe just change this to "History Retention for Physical rReplication of %s" and remove the conditional?  this would be accurate regardless of whether it was a producer or "dummy" and we probably don't want to use "dummy" in public-facing copy.

Reply:
Done.